dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtilityTest.java

```diff
@@ -101,7 +97,7 @@ public void testReadBom_path() throws MavenRepositoryException, ArtifactDescript
         message += removedMessage;
       }
       
-      if (!added.isEmpty() || ! removed.isEmpty()) {
+      if (!added.isEmpty() || !removed.isEmpty()) {
```

Comment:
Doesn't at least one of these have to be true, since the current size is not 219?

Reply:
makes sense, done